Changelog.md

```diff
@@ -1,3 +1,95 @@
+### 0.8.0 (2021-05-07)
+
```

Comment:
Since we make several breaking changes in this release. We should have a `Breaking Changes:` section to specify these changes.